### PR TITLE
Doc 185 document how to set up customer managed

### PR DESCRIPTION
**Pull request by oojo12**
# What does this PR do? 
At this point in time we only have documentation for how to set up Grid-managed BYOC. This version of BYOC works but requires increased permissions. For enterprise users, it will be beneficial for us to have documentation on how to accomplish the same with reduced permissions (Customer-managed BYOC mode). This PR adds instruction for customer-managed BYOC mode to the existing BYOC documentation.

### CODE REVIEW

**Comment by oojo12** [comment]
Had to move from the old repo to this one due to the docs transition. Please re-review + approve if necessary.

**Comment by oojo12** [comment]
@dmitsf I addressed your comment in this version.

**Comment by oojo12** [comment]
In a separate PR we can move to make separate pages for Grid managed vs Customer managed BYOC options if this layout is confusing to anyone.

File: docs/platform/custom-cloud-credentials/adding-custom-cloud-credentials.md [hunk 1]
```diff
@@ -26,11 +92,6 @@ Grid will create clusters designed for large AI workloads. In order to do so, yo
 | EC2 Spot \(instance family you are interested in\) | 1000+ |
 | EC2 On-demand \(instance family you are interested in\) | 1000+ |
 
-AWS STS regional endpoints have to be enabled in the target region. Go to https://console.aws.amazon.com/iam/home#/account_settings and verify the regional endpoint is activated. In most cases your region already has AWS STS regional endpoint enabled -> https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
```
Comment (nmiculinic) [inline docs/platform/custom-cloud-credentials/adding-custom-cloud-credentials.md:29]: Why was this deleted? This is pretty tricky bug to debug.

**Review by nmiculinic — CHANGES_REQUESTED** [review]

**Comment by oojo12** [comment]
closing in favor of the modular PR's that facilitate the BYOC page redesign.

**Pull request closed without merging**